### PR TITLE
Adds a docs folder with some minimal info about crates and commands

### PR DESCRIPTION
**Pull request by felipesere**

### CODE REVIEW

**Comment by felipesere** [comment]
I've added to an issue on cargo because as far as I can tell, `exclude` doesn't really work the way I expected it.
https://github.com/rust-lang/cargo/issues/6745

File: docs/COMMANDS.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Commands
+
+This is where you'll find the implementation of the `oro` commands.
+The commands are meant to feel familiar to users of `npm`, `yarn`, the entropic client `ds`, `tink`, and others.
+
+There are three parts to commands:
+
+ * The datastructure with `clap` annotations that is used for command line arguments.
+ * The implementation of `OroCommand` that contains the actual exectuion of the commands.
```
Comment (laysauchoa) [inline docs/COMMANDS.md:9]:
```suggestion
 * The implementation of `OroCommand` that contains the actual execution of the commands.
```

File: docs/COMMANDS.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Commands
+
+This is where you'll find the implementation of the `oro` commands.
+The commands are meant to feel familiar to users of `npm`, `yarn`, the entropic client `ds`, `tink`, and others.
+
+There are three parts to commands:
+
+ * The datastructure with `clap` annotations that is used for command line arguments.
```
Comment (laysauchoa) [inline docs/COMMANDS.md:8]:
```suggestion
 * The data structure with `clap` annotations that is used for command line arguments.
```

**Comment by felipesere** [comment]
Thank you 💜

**Pull request merged**